### DISCUSSION TITLE
when excute "forwar search" in tex file, it shows "unknown source files"

### DISCUSSION THREAD

**Discussion by DRjy** [James-Yu/LaTeX-Workshop#3156] [Q&A]

**Comment by tamuratak** [comment]
Fill out all the information required in the issue template. Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

**Comment by tamuratak** [comment]
https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md

## Preliminary questions [Required]

### Disable all the other extensions except for LaTeX Workshop, and check that you still see this issue. [Required]

You still see this issue?: Yes/No

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes/No

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes/No




## Describe the bug [Required]

A clear and concise description of what the bug is.

### To Reproduce

Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. See error

### Expected behavior

A clear and concise description of what you expected to happen.




## Logs [Required]

Please paste the whole log messages here, not parts of ones. The log should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

### LaTeX Workshop Output [Required]

```
To access the log, click the 'TeX' icon on the Activity Bar on the left side, select 'View Log Messages', then select 'View LaTeX Workshop extension log'.

[Paste the log here]
```

### Developer Tools Console [Required]

```
To access the log, click 'help' -> 'Toggle Developer Tools' -> 'Console'. Paste anything suspicious.

[Paste the log here]
```



## Screenshots

If applicable, add screenshots to help explain your problem.




## Desktop [Required]
 - OS: [e.g. Windows 10]
 - VS Code version: [e.g. 1.41.0]
 - Extension version: [e.g. 8.5.0]
 - TeX distribution version: [e.g. TeX Live 2019/MiKTeX 2.9.7250]




## Additional questions

### Are you using LaTeX Workshop with VS Code Remote?

Yes/No

If the answer is Yes, please write which one you are using. Write the versions of the remote extension.

- Remote Containers/Remote SSH/Remote WSL
- Remote Extension Version: [e.g. 0.48.0]

### Additional context

Add any other context about the problem here.

**Comment by tamuratak** [comment]
Fill out all the information required in the issue template.

**Comment by tamuratak** [comment]
External PDF viewers are not officially supported. Please use the internal PDF viewer.
